rdr_service/offline/genomic_pipeline.py

```diff
@@ -157,6 +157,19 @@ def aw3_wgs_manifest_workflow():
             load_awn_manifest_into_raw_table(manifest['file_path'], "aw3")
 
 
+def cvl_w1il_manifest_workflow(cvl_site_bucket_map):
+    for site_id in config.GENOMIC_CVL_SITES:
+        cvl_bucket = cvl_site_bucket_map[site_id]
+
+        with GenomicJobController(
+            GenomicJob.CVL_W1IL_WORKFLOW,
+            bucket_name=cvl_bucket,
+            cvl_site_id=site_id
+        ) as controller:
+            controller.generate_manifest(GenomicManifestTypes.CVL_W1IL_PGX, _genome_type=config.GENOME_TYPE_WGS)
+            controller.generate_manifest(GenomicManifestTypes.CVL_W1IL_HDR, _genome_type=config.GENOME_TYPE_WGS)
```

Comment:
Can we split these into two separate jobs so they can be run separately? I can see a case where we get a request to run only a PGx or HDR manifest, or perhaps change the job frequency for one and not the other since PGx is much faster for the CVLs to process.

Reply:
Will do 😄